adapters/repos/schema/store.go

```diff
@@ -21,7 +21,7 @@ import (
 	"path"
 
 	"github.com/sirupsen/logrus"
-	clusterStore "github.com/weaviate/weaviate/cluster/store"
+	clusterSchema "github.com/weaviate/weaviate/cluster/schema"
```

Comment:
**suggestion:** can't we replace `clusterSchema` to `cluster` ? to have 1 word instead of aliasing